main.go

```
@@ -959,13 +973,28 @@ func handleDueCommand() {
+		func(g Goal) string { return g.Baremin },
+		func(g Goal) int64 { return g.Losedate },
+	)
 }
 
 // handleFilteredCommandWithBaremin is like handleFilteredCommand but lets the
-// caller customise the baremin string displayed for each goal. This is used by
-// the tomorrow view to bump due-today goals by one day's rate.
+// caller customise the baremin string displayed for each goal. The deadline
+// shown for each goal still comes from the goal's own losedate. Kept for
+// backwards compatibility with callers that only need to override baremin.
 func handleFilteredCommandWithBaremin(filterName string, filter func(Goal) bool, bareminFor func(Goal) string) {
```

golangci-lint (2.12.2)

[error] func handleFilteredCommandWithBaremin is unused

(unused)